client/src/ui/modules/Achievements.tsx

```diff
@@ -0,0 +1,96 @@
+import { Alert, AlertDescription, AlertTitle } from "@/ui/elements/alert";
+import { Trophy, Star } from "lucide-react";
+import { motion } from "framer-motion";
+import { Game } from "@/dojo/game/models/game";
+import { useTheme } from "@/ui/elements/theme-provider";
+import getElementStyle from "../theme/GetElementStyle";
+
+interface AchievementsProps {
+  gamesByThresholds: { threshold: number; games: Game[] }[];
+  combosByThresholds: { threshold: number; games: Game[] }[];
+}
+
+const AchievementAlert: React.FC<{
```

Comment:
Can be worth to put AchevementAlert elsewhere

Reply:
Yes, created new compo for it